pkg/inspector/image-inspector.go

```diff
@@ -399,12 +399,15 @@ func (i *defaultImageInspector) scanImage(s openscap.Scanner) ([]byte, []byte, e
 		return []byte(""), []byte(""), fmt.Errorf("Unable to read %s result file: %v\n", s.ScannerName(), err)
 	}
 
-	htmlScanReport, err := ioutil.ReadFile(s.HTMLResultsFileName())
-	if err != nil {
-		return []byte(""), []byte(""), fmt.Errorf("Unable to read %s HTML result file: %v\n", s.ScannerName(), err)
+	if i.opts.OpenScapHTML {
+		htmlScanReport, err := ioutil.ReadFile(s.HTMLResultsFileName())
+		if err != nil {
+			return []byte(""), []byte(""), fmt.Errorf("Unable to read %s HTML result file: %v\n", s.ScannerName(), err)
+		}
+		return scanReport, htmlScanReport, nil
```

Comment:
Can we set `htmlScanReport` to nil at the top and here to what it should be to stay with one successful return from the function?